shared/constants/teams.js

```diff
@@ -49,14 +49,16 @@ export type InviteToTeamByPhone = NoErrorTypedAction<
     teamname: string,
     role: string,
     phoneNumber: string,
+    fullName: ?string,
   }
 >
 
 // username -> removeMember
 // email -> removePendingInvite
+// id -> removePendingSeitanInvite
 export type RemoveMemberOrPendingInvite = NoErrorTypedAction<
   'teams:removeMemberOrPendingInvite',
-  {name: string, username: string, email: string}
+  {name: string, username: string, email: string, id: string}
```

Comment:
changing `removeMemberOrPendingInvite` to always use `inviteID` for invites would make this typing better, but core support hasn't been thoroughly tested for non-seitan invites (cc @zapu)